CollAction/Controllers/ProjectsController.cs

```diff
@@ -161,9 +161,15 @@ public async Task<IActionResult> Create(CreateProjectViewModel model)
             foreach (var admin in administrators)
                 _emailSender.SendEmail(admin.Email, subject, confirmationEmailAdmin);
 
-            return View("ThankYouCreate", new ThankYouCreateProjectViewModel()
+            return LocalRedirect($"~/Projects/Create/{project.Id}/{Uri.EscapeDataString(project.Name)}/thankyou");
```

Comment:
I think one of the results of the discussion were that we could make the url more ["user-friendly"](https://collaction.atlassian.net/browse/CA-388?focusedCommentId=14619&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-14619). 

Should be a simple function:
* Strip diacritics: https://stackoverflow.com/questions/249087/how-do-i-remove-diacritics-accents-from-a-string-in-net
* Replace spaces with dashes

Also, don't you need to add the name parameter to Details for this to work, or is that not necessary?